cypress/fixtures/docs/staticJSONObjects.md

```diff
@@ -0,0 +1,144 @@
+# Static JSON Objects
```

Comment:
There are 2 types of JSON objects used in test cases. One is for making firebolt calls to the device and second is for validating the response of the call. 

replace line 3 with this

Reply:
done